app/main.py

```diff
@@ -1,8 +1,50 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def needs_washing(self, car: Car) -> bool:
+        return car.clean_mark < self.clean_power
+
+    def calculate_washing_price(self, car: Car) -> float:
+        if car.clean_mark >= self.clean_power:
+            return 0.0
+        comfort_factor = car.comfort_class
+        cleaning_required = self.clean_power - car.clean_mark
+        rating_factor = self.average_rating
+        location_factor = self.distance_from_city_center
+        price = ((comfort_factor * cleaning_required * rating_factor)
+                 / location_factor)
+        return round(price, 1)
```

Comment:
This is a long math equation. Consider breaking it up into additional variables for better readability, as per the first item in the Code Style section of the checklist. Here is a suggestion: 

python
comfort_factor = car.comfort_class
cleaning_required = self.clean_power - car.clean_mark
rating_factor = self.average_rating
location_factor = self.distance_from_city_center
price = (comfort_factor * cleaning_required * rating_factor) / location_factor
```

This makes the code easier to read and understand.